OpenCeedlingFile.py

```diff
@@ -1,71 +1,84 @@
-import sublime
+from __future__ import print_function
+import os
+import re
+
 import sublime_plugin
-import re, inspect, os
+
 
 class OpenCeedlingFileCommand(sublime_plugin.WindowCommand):
+    def run(self, option):
+        if not self.window.active_view():
+            return
+
+        self.views = []
+        window = self.window
+        current_file_path = self.window.active_view().file_name()

```

Comment:
This may yield `None` for unsaved views and lead to exceptions further down.

Reply:
Noted.